client/homescreen/stats-overview/style.scss

```diff
@@ -4,6 +4,8 @@ $promo-actions-border-top: $core-grey-light-500;
 $promo-actions-border-bottom: $light-gray-tertiary;
 
 .woocommerce-stats-overview {
+	margin-bottom: $gap-large;
```

Comment:
Should this spacing be used across all cards on the homescreen?  Maybe this margin should be moved to `.woocommerce-homescreen-card`.

Or better yet, we could apply this to any root item added to a homescreen column:

```scss
.woocommerce-homescreen-column > div {
	margin-bottom: $gap-large;
}
```

Then we can remove component specific styles like the same bottom margin added to `.woocommerce-homescreen .woocommerce-task-dashboard__container`.

Reply:
yea that would probably be better :)